tap_deputy/utils.py

```diff
@@ -0,0 +1,28 @@
+from typing import Dict
+import json
+from singer import get_logger
+
+LOGGER = get_logger()
+
+def read_config(config_path) -> Dict:
+    """
+    Performs read on the provided filepath,
+    returns empty dict if invalid path provided
+    """
+    try:
+        with open(config_path,'r') as tap_config:
+            return json.load(tap_config)
+    except FileNotFoundError as _:
+        LOGGER.fatal("Failed to load config in dev mode")
+        return {}
+
+
+def write_config(config_path,data :Dict) -> Dict:
```

Comment:
Please be consistent. If you are going to type `data`, why not type `config_path` too?